setup.py

```diff
@@ -16,20 +16,15 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from setuptools import setup
+from distutils.util import convert_path
 
-import imp
+version = {}
+exec(open(convert_path('pycoast/version.py')).read(), version)
```

Comment:
@mraspaud I went back and forth on this. `imp` is deprecated as of python 3.4 (but still available). The preferred way is using `importlib` as a replacement, but from all my attempts this requires importing `pycoast/__init__.py` (the package). Doing so would import all modules imported in the init module which will fail since the package isn't installed yet (in some environments).

The other non-import options are:

1. Use regular expressions or similar to parse the text of the file.
2. Use a source code importer in importlib. Requires 3 or more source code lines and looked ugly to me so I wasn't a huge fan.
3. Use a third party package to handle it for us (but essentially does the same thing).

I'll check again if setuptools has something for doing this automatically.